submissions/ksalpern/Hooli-style-Popup/index.html

```diff
@@ -0,0 +1,275 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Hooli-style-Popup</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+        <div class="header__wrapper">
+          <nav class="header__nav">
+            <ul class="nav__items">
+              <li class="nav__item">
+                <a class="nav__item-link" href="#">Mail</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__item-link" href="#">Images</a>
+              </li>
+
+              <li class="nav__item">
+                <input id="checkbox" class="checkbox" type="checkbox" />
+                <label for="checkbox">
+                  <img
+                    class="popup__button"
+                    src="./popup-icons/popup-button.png"
+                    alt="Popup button"
```

Comment:
Read [this article ](https://htmlacademy.ru/blog/articles/alt-text) about how and when to populate `alt` attribute.